hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/CompactTestUtil.java

```diff
@@ -59,12 +59,12 @@ static GenericRecord createCompactGenericRecord(MainDTO mainDTO) {
         }
 
         GenericRecord innerRecord = GenericRecordBuilder.compact("inner")
-                .setArrayOfInt8("b", inner.bb)
-                .setArrayOfInt16("s", inner.ss)
-                .setArrayOfInt32("i", inner.ii)
-                .setArrayOfInt64("l", inner.ll)
-                .setArrayOfFloat32("f", inner.ff)
-                .setArrayOfFloat64("d", inner.dd)
+                .setArrayOfInt8("b", inner.bytes)
+                .setArrayOfInt16("s", inner.shorts)
+                .setArrayOfInt32("i", inner.ints)
+                .setArrayOfInt64("l", inner.longs)
+                .setArrayOfFloat32("f", inner.floats)
+                .setArrayOfFloat64("d", inner.doubles)
```

Comment:
We should also set inner.strings

Reply:
Also I think one of the tests should fail if this is the case 😅, should we add such a test?

Reply:
This is a utility method to something else with GenericRecords. I think indication of this is missing should be code coverage not a failing test. I am adding the missing call to setArrayOfStrings